### ISSUE TITLE
NCAP 0.99-r7 failed to setup on Win server 2012 2

### ISSUE THREAD

**Issue by mduchaff** [nmap/nmap#1307]
Hello, 
I tried to install nmap7.70. It tried to install npcap0.99-r2. It ended with a  message saying: 
"unknown error! 4a020 NPCAP LWF driver 'with Wi-Fi support) has failed to be installed.
Failed to create the npcap service for Win7, Win8 and Win10"
I downloaded and installed npcap0.99-r7 with the same problem
Thanks for any help.
Marc
attached : npcap0.99-r7 install.log
[install.log](https://github.com/nmap/nmap/files/2305652/install.log)

**Comment by mduchaff** [comment]
When I ran C:\Program Files\Npcap\DiagReport.bat, part of the result was:
My translation :"impossible to find the access path " HKLM:\..." because it dos not exist

*************************************************
WinPcap Info:
*************************************************
HKLM:\SOFTWARE\WOW6432Node\WinPcap:
Get-ItemProperty : Impossible de trouver le chemin d'accès « HKLM:\SOFTWARE\WOW6432Node\WinPcap », car il n'existe pas.
Au caractère C:\Program Files\Npcap\DiagReport.ps1:170 : 6
+     (Get-ItemProperty HKLM:\SOFTWARE\WOW6432Node\WinPcap | out-string -stream |  ...
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (HKLM:\SOFTWARE\WOW6432Node\WinPcap:String) [Get-ItemProperty], ItemNotF 
   oundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetItemPropertyCommand

**Comment by mduchaff** [comment]
The problem arises when I asked for the Wifi option. I went around in letting this install option out.

**Comment by rugabunda** [comment]
Looks like I have figured out the culprit in my circumstance. This results when the npcap service & certain npcap drivers fails to install, the npcap loopback interface then fails to install, resulting in problems such as this. My driver finally succeeded installing after temporarily disabling group policy `Admin Templates\System\Device Installation\Device Installation Restrictions\Prevent Installation of devices using drivers that match these device setup classes\ (show) {D48179BE-EC20-11D1-B6B8-00C04FA372A7}`

This class of driver was specifically blocked from installing, which is disabled as part of windows in built USB DMA attack protection for bitlocker, via Thunderbolt & SBP2 devices: https://support.microsoft.com/en-us/help/2516445/blocking-the-sbp-2-driver-and-thunderbolt-controllers-to-reduce-1394-d

    IEEE 1394 Devices That Support the SBP2 Protocol Class = SBP2 ClassGuid = {d48179be-ec20-11d1-b6b8-00c04fa372a7}

Strangely, even after re-enabling this policy, the installation problem doesn't seem to be persisting; Why? I don't know. This was the first time in months I was able to install Npcap. The only other policy and or setting I had changed was setting group policy enabled  `"Boot-Start Driver Initialization Policy" to "Good only".`
